packages/ui/src/Utils.ts

```
@@ -0,0 +1,151 @@
+    }
+    listeningEntities.length = count;
+  }
+
+  private static _unRegisterListener(listener: (flag: number, param?: any) => void, listeningEntities: Entity[]): void {
+    for (let i = 0, n = listeningEntities.length; i < n; i++) {
+      // @ts-ignore
+      listeningEntities[i]._unRegisterModifyListener(listener);
+    }
+    listeningEntities.length = 0;
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)